OPI_RELEASE_APPROACH.md

```diff
@@ -0,0 +1,85 @@
+# OPI Release Approach
+
+OPI will release some artifacts around the work being done in the
+project. This document details *how* the artifacts are released, and how it
+relates to the form the project takes, meaning this document is the result of a
+[mailing list discussion](https://lists.opiproject.org/g/tsc/message/30)
+around the OPI release process.
+
+## Concrete OPI Release Approach
+
+OPI will be releasing a set of artifacts, including a reference architecture
```

Comment:
I feel like this paragraph is redundant, lets start with the actual list

Reply:
I disagree and I will leave this sentence in here.